File: DemoApp/DemoApp/NativePlusView.m
//
//  NativePlusView.m
//  DemoApp
//
//  Created by Glispa GmbH on 2/10/16.
//  Copyright © 2016 Glispa GmbH All rights reserved.
//

#import "NativePlusView.h"


@implementation NativePlusView

+ (CGFloat)desiredHeight {
    return 300;
}


#pragma mark - Overriden methods


// Implement + (NSString*)xibName only if this class is related with XIB
+ (NSString *)xibName {
    return NSStringFromClass([self class]);
}


@end

File: DemoApp/DemoAppSwift/NativePlusView.swift
//
//  NativePlusView.swift
//  DemoApp
//
//  Created by Glispa GmbH on 19/05/16.
//  Copyright © 2016 Glispa GmbH. All rights reserved.
//

import UIKit
import AvocarrotNativeView

class NativePlusView: AVONativeView {

    static func desiredHeight() -> CGFloat {
        return 300.0
    }

    override static func xibName() -> String {
        return NSStringFromClass(self)
    }

}
